.gitignore

```diff
@@ -7,4 +7,5 @@
 /pkg/
 /spec/reports/
 /tmp/
-*.gem
\ No newline at end of file
+*.gem
+/.idea/
```

Comment:
You mean out of hand like this one? https://github.com/usertesting/orders/blob/get-v2-api-working/.gitignore

BTW that one also includes this ignore statement.